API/src/models/parcel.js

```diff
@@ -0,0 +1,22 @@
+// Class for defining a parcel model
+class Parcel {
+  constructor(id, name, origin, destination, weight, userId) {
+    this.id = id;
+    this.name = name;
+    this.origin = origin;
+    this.destination = destination;
+    this.weight = weight;
+    this.userId = userId;
+	    this.setPrice();
+  }
+
+  setPrice() {
+    if (typeof this.weight === 'number') {
```

Comment:
Leverage on javascript's truthy and falsy concepts.
Could be refactored to if (Number(this.weight)) ...